source/about/cloud-subscriptions

```diff
@@ -0,0 +1,116 @@
+===================
```

Comment:
The file extension isn't yet defined for this new page as .rst
Also - will we want this new page surfaced up in the LHS?

Reply:
Fixed by adding the extension. Yes, I think this should be LHS visible.

Reply:
Great! To add new topics to the About section of the docs, you'll add paths to the new pages to ``overview/index.html`` file, like this:

```
.. toctree::
   :maxdepth: 2
   :glob:

   /about/product
   /about/self-hosted-subscriptions
   /about/cloud-subscriptions
   /about/security
```

Reply:
@cwarnermm I've decided to keep it to only the subscription overview for the LHS.  How do I add the "Self-hosted subscriptions" and "Cloud subscriptions" as pages that are nested under the LHS? I believe now its based on headers on the page itself.